spec/predicates/reference.md

```diff
@@ -0,0 +1,143 @@
+# Predicate type: Reference
+
+Type URI: https://in-toto.io/attestation/reference/v0.1
+
+Version: v0.1.0
+
+## Purpose
+
+The reference attestation references a set of documents that are relevant to
+some resource. It is used to point to and sign metadata that is not
+provided in-band.
+
+## Use Cases
+
+One use case is to provide an SBOM of an artifact without needing to sign
+the entire SBOM and transmit it in-band. A consumer, such as
+[GUAC](https://guac.sh/), retrieves the SBOM, verifies the digest, and uses it
+for vulnerability management, compliance, analysis, or any other purpose.
+
+Another use case is to provide evidence for a policy such as "deny unless a
+reference to an SBOM is provided".
+
+There is encodability overlap with some other predicates, mainly SCAI, but
+the use case is distinct.
```

Comment:
Can you say more about why SCAI isn't quite right for this case?  (E.g. can we answer [this question](https://github.com/in-toto/attestation/blob/main/docs/new_predicate_guidelines.md#:~:text=Why%20don%E2%80%99t%20existing%20predicates%20cover%20this%20use%20case%3F))

Reply:
Done.